swagger.yaml

```diff
@@ -6548,6 +6548,8 @@ definitions:
       - 'servicemesh-requirements-satisfied'
       - 'serverless-requirements-satisfied'
       - 'openshift-ai-requirements-satisfied'
+      - 'mtu-test-between-hosts-in-machine-network'
```

Comment:
Why do we need 2 validation ids?

Reply:
We want to separate the validations for CMN and UMN, so that we can disable one while keeping the other enabled, or make one required for installation while the other remains a warning only etc..